extra/1-factorial.js

```diff
@@ -10,6 +10,15 @@
 
 function factorial(input) {
   // TODO
+  if (input === 0 || input === 1) {
+    return 1;
```

Comment:
Here, I think if the input is 0, you should return 0, rather than return 1.
So instead, line 14 could be as follows:  
return input;

Reply:
thank you,i will look into it